mne/transforms.py

```diff
@@ -248,6 +248,47 @@ def rotation3d(x=0, y=0, z=0):
     return r
 
 
+def rotation3d_align_z_axis(target_z_axis):
+    """Compute a rotaion matrix to align [ 0 0 1] with supplied target z axis.
+
+    Parameters
+    ----------
+    target_z_axis : array, shape = (1, 3)
+        z axis. computed matrix (r) will map [0 0 1] to target_z_axis
+
+    Returns
+    -------
+    r : array, shape = (3, 3)
+        The rotation matrix.
+    """
+    target_z_axis = target_z_axis / np.linalg.norm(target_z_axis)
+    r = np.zeros((3, 3))
+    if ((1 + target_z_axis[2]) < 1E-12):
+        r[0, 0] = 1
```

Comment:
please 1. not 1 when quantities are meant to be floats

same remake below and above